README.md

```diff
@@ -157,7 +157,18 @@ possible by the use of the `UserService` service provided by
 4. `return {token}` - send the JWT.
 
 You can see the details in
-[`packages/shopping/src/controllers/user-management.controller.ts`](https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user.controller.ts).
+[`packages/shopping/src/controllers/user-management.controller.ts`](https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user-management.controller.ts).
+
+### JWT secret key
+
+By default, the JWTs will be signed using HS256 with a 256 bit long random
+secret. To use your own secret, set environment variable JWTSECRET to the value
```

Comment:
@johanschvn should probably call the env variable `JWT_SECRET` which makes it easier to read and also follows the general naming convention

Reply:
Agree, as currently used env variable RECOMMENDER_PROTOCOL follow that standard.